client_it_test.go

```diff
@@ -307,6 +307,44 @@ func TestTopNReturns(t *testing.T) {
 	}
 }
 
+func TestMinMaxRow(t *testing.T) {
+	client := getClient()
+	field := index.Field("test-field")
```

Comment:
Can you change the field name to something like `test-minmaxrow-field`? It seems we've used `test-field` elsewhere and that makes this test fail.

Reply:
done!